chart/values.schema.json

```
@@ -142,11 +142,35 @@
                             "default": ""
                         },
                         "hosts": {
-                            "description": "The hostnames for the web Ingress.",
+                            "description": "The hosts configuration for the web Ingress.",
                             "type": "array",
                             "default": [],
                             "items": {
-                                "type": "string"
+                                "type": "object",
+                                "properties": {
+                                    "name": {
+                                        "description": "The hostname for the web Ingress.",
+                                        "type": "string",
+                                        "default": ""
+                                    },
+                                    "tls": {
+                                        "description": "Configuration for web Ingress TLS.",
+                                        "type": "object",
+                                        "additionalProperties": false,
+                                        "properties": {
+                                            "enabled": {
+                                                "description": "Enable TLS termination for the web Ingress.",
+                                                "type": "boolean",
+                                                "default": false
+                                            },
+                                            "secretName": {
+                                                "description": "The name of a pre-created Secret containing a TLS private key and certificate.",
+                                                "type": "string",
+                                                "default": ""
+                                            }
+                                        }
+                                    }
+                                }
```

Probably want to ensure we at least get a host, no?

```suggestion
                                },
                                "required": ["host"],
```